shell/platform/android/android_exports.lst

```diff
@@ -9,11 +9,16 @@
     JNI_OnLoad;
     _binary_icudtl_dat_start;
     _binary_icudtl_dat_size;
-    shorebird_init;
-    shorebird_active_path;
-    shorebird_active_patch_number;
-    shorebird_free_string;
+    
     shorebird_check_for_update;
+    shorebird_free_string;
+    shorebird_init;
+    shorebird_next_boot_patch_number;
+    shorebird_next_boot_patch_path;
+    shorebird_report_launch_failure;
```

Comment:
I don't think you want to expose the report_ symbols to Dart code?  Also not sure about the start_update_thread.  Honestly not even sure we need _init anymore either.

We can expose all these, but I just worry they're footguns.

Reply:
Good call, removed all but the ones that will be used by v1 of the bindings

Reply:
Reintroduced shorebird_free_string to handle the char* returned by `shorebird_next_boot_patch_number`